Test is flaky because of undeterministic ordering

```

FAIL: TestTagCollectionUnique (0.00s) | 2s
-- | --
  | --- FAIL: TestTagCollectionUnique/less_specific_patch_version (0.00s)
  | require.go:157:
  | Error Trace:	version_test.go:110
  | Error:      	Not equal:
  | expected: []string{"0.1.0", "0.1"}
  | actual  : []string{"0.1", "0.1.0"}
  |  
  | Diff:
  | --- Expected
  | +++ Actual
  | @@ -1,4 +1,4 @@
  | ([]string) (len=2) {
  | - (string) (len=5) "0.1.0",
  | - (string) (len=3) "0.1"
  | + (string) (len=3) "0.1",
  | + (string) (len=5) "0.1.0"
  | }
  | Test:       	TestTagCollectionUnique/less_specific_patch_version
  | FAIL


```

--- a/pkg/outdated/version_test.go
+++ b/pkg/outdated/version_test.go
@@ -316,7 +316,7 @@ func TestRemoveLeastSpecific(t *testing.T) {
 			tags := collection.RemoveLeastSpecific()
 
 			originalRemoved := makeOriginal(tags)
-			require.Equal(t, test.expectVersions, originalRemoved)
+			require.EqualValues(t, test.expectVersions, originalRemoved)
 		})
 	}
 }

